src/utils/getUserBy.js

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
+const preparedUsers = [...usersFromServer];
+
+export const getUserBy = todo =>
+  preparedUsers.find(user => user.id === todo.userId);
```

Comment:
Ensure that the `todo` object passed to `getUserBy` always has a valid `userId` property to avoid unexpected behavior.